src/commands/object/gen_helper.ts

```diff
@@ -0,0 +1,152 @@
+import * as graphene from "graphene-pk11";
```

Comment:
No copy pasta.
Test generate and Generate should use the same helper.
You'll have to refactor the helper and test command a bit.